canonical/maas-ui
# refactor: remove dead code

## Done
- refactor: remove dead code

This removes some of the dead code as identified by `npx unimported`

## QA
- Verify that machine list, machine details, controller -> networking tab load without errors
